text/0055-nested-multi-sigs.md

```diff
@@ -1,35 +1,35 @@
-- Feature Name: nested_multi_sig
+- Feature Name: Nested Minting Multi-Signatures
```

Comment:
:upside_down_face: With my better understanding and now reading my suggestion it seems like this is "minting" "multiple signatures"
```suggestion
- Feature Name: Nested Signer Sets for Minting
```